functions-methods.js

```diff
@@ -1,36 +1,83 @@
+        }
+    }
+    return domain;
+}
+let domain = getEmailDomain("mazilemka@gmail.com");
 
-
+console.log("Opdracht 1: " + domain);
 
 
 /* Opdracht  2 */
-// Schrijf een functie genaamd typeOfEmail, die een emailadres verwacht. De functie checkt of het emailadres een novi domein heeft (medewerker), een novi-education domein (student), of extern domein (zoals gmail of outlook)
+// Schrijf een functie genaamd typeOfEmail, die een emailadres verwacht.
+// De functie checkt of het emailadres een novi domein heeft (medewerker),
+// een novi-education domein (student), of extern domein (zoals gmail of outlook)
 // ---- Verwachte uitkomsten:
 // typeOfEmail("n.eeken@novi-education.nl") geeft "Student"
 // typeOfEmail("t.mellink@novi.nl") geeft geeft "Medewerker"
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function typeOfEmail(eMail) {
+    let domain = getEmailDomain(eMail);
+    switch (domain) {
+        case "novi.nl":
+            return "Medewerker";
+            break;
+        case "novi-education.nl" :
+            return "Student";
+            break;
+        default :
+            return "Extern";
+    }
+}
+console.log("\nOpdracht 2: ");
+console.log(typeOfEmail("t.mellink@novi.nl"));
+console.log(typeOfEmail("n.eeken@novi-education.nl"));
+console.log(typeOfEmail("novi.nlaapjesk@outlook.com"));
+console.log(typeOfEmail("a.wiersma@outlook.com"));
 
 
 /* Opdracht  3 */
-// Schrijf een functie genaamd checkEmailValidity, die een emailadres verwacht en checkt of het emailadres valide is. De functie returned true of false, afhankelijk van de uitkomst.
+// Schrijf een functie genaamd checkEmailValidity, die een emailadres verwacht en checkt of
+// het emailadres valide is. De functie returned true of false, afhankelijk van de uitkomst.
 // Een emailadres is valide wanneer:
 // * Er een @ in voorkomt
 // * Er géén , in voorkomt
-// * Er géén . in voorkomt als allerlaatste karakter (dus hotmail.com is valide, net als outlook.nl, maar outlooknl. niet)
+// * Er géén . in voorkomt als allerlaatste karakter (dus hotmail.com is valide, net als outlook.nl,
+// maar outlooknl. niet)
 // ---- Verwachte uitkomsten:
 // checkEmailValidity("n.eeken@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(eMail) {
+    if (eMail.includes('@') && !eMail.includes(',') && !(eMail.charAt(eMail.length - 1) === '.')) {
+    return true;
+} else return false;
```

Comment:
Dit kun je nog een klein beetje versimpelen
```suggestion
    return eMail.includes('@') && !eMail.includes(',') && eMail.charAt(eMail.length - 1) !== '.';
```